### PR TITLE
[#27]/[#30] Use Pubmed ID If No DOI For Altmetrics / Altmetrics Badge Bug

### PR DESCRIPTION
**Pull request by Meowcenary**
If there is no DOI, try to use Pubmed ID for altmetrics. Resolve issue where Altmetrics badge would disappear from views.

### CODE REVIEW

**Comment by Meowcenary** [comment]
Test files to view:

Working Author card: https://search.library.northwestern.edu/discovery/fulldisplay?docid=alma9957487804202441&con[…]erything&query=any,contains,michelle%20obama&offset=0
Working Altmetrics with DOI: https://search.library.northwestern.edu/discovery/fulldisplay?docid=cdi_crossref_primary_10_[…]sky%20and%20Education%20through%20Punishment&offset=0
Working Altmetrics with PMID: https://search.library.northwestern.edu/discovery/fulldisplay?docid=cdi_proquest_miscellaneo[…]ain%20Tumor%20with%20%5Bmethyl-11C%5DCholine&offset=0 (edited)

File: primo-explore/custom/main.js [hunk 1]
```diff
@@ -515,8 +515,20 @@ app.controller('FullViewAfterController', ['angularLoad', '$http', '$scope', '$e
   {
     vm.parentElement = this.$element.parent()[0]; //the prm-full-view container
     try {
-        vm.doi = vm.parentCtrl.item.pnx.addata.doi[0] || '';
+      console.log(vm.parentCtrl.item.pnx.display);
+      console.log(vm.parentCtrl.item.pnx.addata);
```
Comment (sharpattack) [inline primo-explore/custom/main.js:519]: Quite a few `console.log` added. Can we remove some, or are they all required?

**Review by sharpattack — APPROVED** [review]
Minor question, but looks great. Thank you for providing examples.

File: primo-explore/custom/main.js (hunk 1 above)
Comment (Meowcenary) [reply primo-explore/custom/main.js:519]: Oh, I will remove those.

**Comment by jpastva** [comment]
Thanks @Meowcenary, this looks great!

**Pull request merged**